### DISCUSSION TITLE
save question

### DISCUSSION THREAD

**Discussion by HPL001** [idaholab/moose#29628] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Dear moose Community,
 Hello, I have a question about saving. I output the following Settings, I can output.e files normally before, but I suddenly found that I output a series of such files,.e files only save the result of the first time step. In addition, I can view it normally in peacock.
![image](https://github.com/user-attachments/assets/05aeffb0-4a40-404f-b558-22d23d762236)
![image](https://github.com/user-attachments/assets/0dfe22e9-c112-4e53-98aa-4c2065a081b2)

**Comment by GiudGiud** [comment, marked as answer]
Paraview will understand these -s00x extensions 

This is due to using either adaptivity or mesh modifications

**Reply by HPL001** [reply]
Thank you for your guidance
